akka-cluster-sharding-typed/src/main/scala/akka/cluster/sharding/typed/delivery/internal/ShardingProducerControllerImpl.scala

```diff
@@ -279,7 +279,9 @@ private class ShardingProducerControllerImpl[A: ClassTag](
   import ShardingProducerController.Start
   import ShardingProducerControllerImpl._
 
-  private val durableQueueAskTimeout: Timeout = settings.producerControllerSettings.durableQueueRequestTimeout
+  // Chunked messages not implemented for ShardingProducerController (yet)
+  private val producerControllerSettings = settings.producerControllerSettings.withChunkLargeMessagesBytes(0)
```

Comment:
Should there be a warning logged or an error or something so the user does not think they have enabled chunking with sharding?

Reply:
made it illegal setting, fixed in 193fc337be